test/__mocks__/ComponentStub.tsx

```diff
@@ -1,7 +1,5 @@
 import React, { ReactElement } from "react"
 
-const ComponentStub = (): ReactElement => {
+export const ComponentStub = (): ReactElement => {
```

Comment:
Any reason for no `default`?

```suggestion
export default const ComponentStub = (): ReactElement => {
```

If you add `default` then you don't need to add the `{}` around it when importing

Reply:
I don't think that's valid syntax:
![image](https://user-images.githubusercontent.com/43986931/92479235-3191ec00-f1db-11ea-9d15-9dc80e0aafc0.png)

Also I read this which gave some good reasons against default exports:
https://basarat.gitbook.io/typescript/main-1/defaultisbad

Reply:
mb it works with 
```typescript
export default function ComponentStub(): ReactElement {}
```

Those seem good reasons for not default exporting but for a file like this which seems to me is never going to change and is not part of a library, `default` makes things cleaner. @pavsidhu you have an opinion on this?

Reply:
Yeahh I agree with @adamjhc on this, keep main components as default export the same way as in the example above. Also yeah default export is bad but it's generally standard to export components as the default export, we could ignore this but then libraries we'd have to use default imports too so it doesn't really make sense